### DISCUSSION TITLE
Verbose log feature request

### DISCUSSION THREAD

**Discussion by davidscan** [theophanemayaud/video-simili-duplicate-cleaner#130] [Ideas]
FIrst off, great job.  I just picked it up off the Apple store and it's easily the best dupe finder I've tried.  Totally worth the $.  One request and one question...I've only used this for a couple days, so maybe I've missed something obvious.

Could you add to the verbose log the file that is "kept" in addition to the file that's "moved"?  That'd make an easy high-level sanity check after running an auto-remove.  

Would it be practical/possible to give the option of enabling ffmpeg's hardware acceleration modes?  I have no idea whatsoever if this would be useful in saving time.  I would call this more of a question than a request.

Thanks!

**Comment by theophanemayaud** [comment]
Hey @davidscan ! Thanks a lot for the suggestions. Sorry I was a bit delayed, but I just released v1.11.0 on macOS which adds more logging like you asked, when verbose is enabled. It now even logs to a file if that’s more convenient for you. 

As for the hardware acceleration, I looked at it some time ago but the app isn’t doing one intensive task, rather «lightly skimming» across multiple videos. Hardware acceleration gives improvements for example when reencoding videos which is intensive and long. But in the case of this app, there wasn’t much to gain, so I decided not to add the complexity.
